i18n/locales/en.json

```diff
@@ -125,6 +125,8 @@
     },
     "modEditSubmissionTopNav": {
         "saveAndExit": "Save & Exit",
+        "saving": "Saving...",
+        "retryMessage": "Something’s not right. Retrying in {seconds}s...",
```

Comment:
I'm a little nervous about the variable being in the translation string here vs just splitting the string into two parts.  Personally, I would change it, but I'll allow it if you prefer this way

Reply:
reason: separate strings prevents any mistakes where a future dev might not be aware there's a variable, or forget to put in in one of the translations which would break the code. Explicit sentence parts makes it apparent what is being built.

Reply:
When it comes to splitting it would it still easily be clear with seconds being separated from the from part. It is just for the s or 秒